frontend/kolide/entities/teams.ts

```diff
@@ -21,8 +21,15 @@ export default (client: any) => {
       const endpoint = `${client._endpoint(TEAMS)}/${teamId}`;
       return client.authenticatedDelete(endpoint);
     },
+    load: (teamId: number) => {
+      const { TEAMS } = endpoints;
+      const endpoint = client._endpoint(`${TEAMS}/${teamId}`);
 
-    loadAll: (page = 0, perPage = 100, globalFilter = "") => {
+      return client
+        .authenticatedGet(endpoint)
+        .then((response: any) => response.team);
```

Comment:
```suggestion
        .then((response: ITeamsResponse) => response.team);
```
Does this work?

Reply:
Ive added a different type for the response. We are actually not yet using `load` method, but i remember using it once and it returned a response like this ```{team: {...} }``` so the ITeamsResponse type wouldnt work as it is ```{teams: [...]}```